transactions/types.go

```diff
@@ -89,3 +89,10 @@ func RPCCalltoSendTxArgs(args ...interface{}) (SendTxArgs, error) {
 
 	return txArgs, nil
 }
+
+// UnmarshalSendTxRPCParams creates SendTxArgs based on RPC JSON
+func UnmarshalSendTxRPCParams(rpcParamsJSON string) (SendTxArgs, error) {
```

Comment:
This is just a wrapper that runs `json.Unmarshal` and is used only once in `lib/library.go`. I would keep it on the caller side (i.e. in `lib/` instead).